src/EnrollCallAndSmsController.js

```diff
@@ -24,11 +27,13 @@ define([
 
   var _ = Okta._;
   var { Keys } = Okta.internal.util;
-  var warningTemplate = '<div class="okta-form-infobox-warning infobox infobox-warning login-timeout-warning">\
+  var warningTemplate = Okta.View.extend({
+    template: hbs('<div class="okta-form-infobox-warning infobox infobox-warning login-timeout-warning">\
```

Comment:
nit:
shall we follow to avoid any DOM changes
```
View.extend({
  className: 'okta-form-infobox-warning....',
  template: hbs('<span...></span><p>..</p>')
})

Reply:
I'm not exactly sure how that works, this is related to how courage is implemented?

Reply:
yeah.
`View` will add another parent (`div` by default) around the `template`